src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/GoToShooterAngle.java

```diff
@@ -0,0 +1,13 @@
+package edu.greenblitz.robotName.commands.shooter.shootingState;
+
+import edu.greenblitz.robotName.commands.shooter.pivot.MovePivotToAngle;
+import edu.greenblitz.robotName.commands.shooter.pivot.PivotCommand;
+import edu.greenblitz.robotName.shootingStateService.ShootingState;
+
+public class GoToShooterAngle extends PivotCommand {
+
+    @Override
+    public void execute() {
+        new MovePivotToAngle(ShootingState.getTargetShooterAngle().getDegrees());
+    }
+}
```

Comment:
1. this doesn't do anything
2. it extends pivot command so it requires the pivot
3. if this did work it would create a billion MovePivotToAngle commands

make this work by maybe putting it in initialize and extend GBCommand
or extend MovePivotToAngle